tests/lib/assertions/ubuntu-core-20-amd64.model

```diff
@@ -0,0 +1,46 @@
+type: model
+authority-id: vxj7EkYOlg15uAHRswTXIUpzWqO2sTBi
```

Comment:
we need this to be a canonical model in some form

Reply:
I added a note to our foundations sync to create a dangerous model.